traveler-ios-ui/TravelerKitUI/ViewControllers/CancelOrderViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CancelOrderViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+protocol CancelOrderViewControllerDelegate {
+    func orderCancelSucceed(with order:Order)
+}
+
+class CancelOrderViewController: UITableViewController {
+    @IBOutlet weak var totalRefundLabel: UILabel!
+
+    internal var quote: CancellationQuote?
+    internal var delegate:CancelOrderViewControllerDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        guard let quote = quote else {
+            Log ("No quote", data: nil, level: .error)
+            return
+        }
```

Comment:
This is ok, but I usually like to not do guard checks if I can get away with optional chaining...makes code lighter